src/Reader/Feed/Atom.php

```diff
@@ -143,9 +143,9 @@ public function getDateModified()
     /**
      * Get the feed lastBuild date. This is not implemented in Atom.
      *
-     * @return null|string
+     * @return void
      */
-    public function getLastBuildDate()
+    public function getLastBuildDate(): void
```

Comment:
Please do not add php typehints as this will is a BC break unless the class is final.